astrocut/cube_cut.py

```diff
@@ -440,7 +446,7 @@ def _get_cutout(self, transposed_cube, verbose=True):
         return img_cutout, uncert_cutout, aperture
 
 
-    def _update_primary_header(self, primary_header):
+    def _update_primary_header(self, product, primary_header):
```

Comment:
`product` parameter needs documentation

Reply:
(There are more places in the code where this has to happen. I'm not going to mark them all.)